Emby.Dlna/Api/DlnaServerService.cs

```diff
@@ -152,12 +157,17 @@ public object Get(GetDescriptionXml request)
             var cacheKey = Request.RawUrl.GetMD5();
             var bytes = Encoding.UTF8.GetBytes(xml);
 
-            return _resultFactory.GetStaticResult(Request, cacheKey, null, cacheLength, XMLContentType, () => Task.FromResult<Stream>(new MemoryStream(bytes)));
+            return _resultFactory.GetStaticResult(Request, cacheKey, null, cacheLength, XMLContentType, () => Task.FromResult<Stream>(new MemoryStream(bytes)));            
```

Comment:
```suggestion
            return _resultFactory.GetStaticResult(Request, cacheKey, null, cacheLength, XMLContentType, () => Task.FromResult<Stream>(new MemoryStream(bytes)));
```

Reply:
VS2019 likes to put blank spaces everywhere. How hard would it be for M$ to fix the editor?